webscripts/build.gradle

```diff
@@ -26,4 +26,13 @@ dependencies {
     compileOnly 'com.google.code.findbugs:jsr305:2.0.1'
 
     compileOnly "javax.annotation:javax.annotation-api:${javaxAnnotationApiVersion}"
+    compileOnly project(':webscripts:webscripts-70')
```

Comment:
Why is there no check for the Alfresco version (`alfrescoBom.startsWith('7')`) ?

Reply:
Because the main `webscripts` are not built against multiple Alfresco versions.

We need to compile against at least one of those version-specific instances to get an `JsonWriterFactoryImpl`